Docs/technical/wflhash.md

```
@@ -2,6 +2,8 @@
 
 # **WFLHASH: A Specification for a High-Performance, Next-Generation Hashing Algorithm**
 
+**SECURITY UPDATE (September 2025):** This document describes the original WFLHASH specification. The implementation has been significantly enhanced with critical security fixes. See the [Security Improvements](#security-improvements) section for details on the current secure implementation.
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)